Cython/Includes/libc/math.pxd

```diff
@@ -104,3 +104,9 @@ cdef extern from "<math.h>" nogil:
     bint isnan(long double)
     bint isnormal(long double)
     bint signbit(long double)
+    int fpclassify(long double)
+    const int FP_NAN
```

Comment:
`const int`, or just `enum`?

Reply:
The values of the `FP_*` constants are implementation defined.
I don't believe an enum should be used here (unless I'm missing something).